Source/Applications/MiMD/wwwroot/Scripts/TSX/MiMD/Configuration/ConfigurationFileRules.tsx

```diff
@@ -279,7 +298,7 @@ const ConfigurationFileRules = () => {
                                 <button type="button" className="btn btn-light float-right" onClick={() => setShowFunctionHelp(true)}>
                                     <i style={{ color: '#007BFF' }} className="fa fa-2x fa-question-circle"></i>
                                 </button>
-                                {showFunctionHelp && <DynamicHelper isOpen={showFunctionHelp} onClose={() => setShowFunctionHelp(!showFunctionHelp)} />}
+                                {showFunctionHelp && <HelperTable Data={help} Title={"Dynamic Expression Examples"} IsOpen={showFunctionHelp} onClose={() => setShowFunctionHelp(!showFunctionHelp)}/>}
```

Comment:
Because you already define `isOpen` you can just render the `HelperTable` regardless.

```suggestion
                                <HelperTable Data={help} Title={"Dynamic Expression Examples"} IsOpen={showFunctionHelp} onClose={() => setShowFunctionHelp(!showFunctionHelp)}/>
```